addons/binding/org.openhab.binding.xmppclient/src/main/java/org/openhab/binding/xmppclient/handler/XMPPClientMessageSubscriber.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.xmppclient.handler;
+
+/**
+ * Subscriber interface
+ * 
+ * @author Pavel Gololobov - Initial contribution
+ */
+public interface XMPPClientMessageSubscriber {
+
+    void processMessage(String from, String payload);
+    String getName();
+    
+}
```

Comment:
newline missing

Reply:
I mean at the end of the file. In github you see this little icon for files without newlines. All files are required to have a newline except the manifest file (which ceases to exist on the new buildsystem though)

Reply:
Ok